feg/gateway/fabfile.py

```diff
@@ -262,7 +262,7 @@ def __init__(
 class S6aConfigs:
     def __init__(
         self,
-        plmn_ids: List[str] = list(),
+        plmn_ids: List[str] = None,
```

Comment:
If it's a list then shouldn't the default value be `[]`? Otherwise I think it should be `Optional[List[str]]`

Reply:
Mutable defaults should always be avoided in Python: https://docs.python-guide.org/writing/gotchas/
I'll add the `Optional` here as well.

Reply:
Actually on second thought, I will not add the `Optional` here since it is used nowhere else in the file. This might eventually be worth doing in a bigger refactoring PR.

Reply:
Sure, sounds good to me.